src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+div {
```

Comment:
div is a block element, block elements are 100% wide

Reply:
changes have been made